circle.yml

```diff
@@ -39,11 +39,8 @@ machine:
 
 checkout:
   post:
-    - |
-      git clone --depth 1 ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
-      cd /home/ubuntu/st2/st2-packages
-      git checkout ${CIRCLE_BRANCH} || true
```

Comment:
As noted before, see the discussion why this order `|| true` hack was needed:
https://github.com/StackStorm/st2/pull/2569